frontend/helpers/airtable.tsx

```diff
@@ -73,3 +92,79 @@ export const getRoommatesData = async () => {
   queryResult.unloadData();
   return roommatesData;
 };
+
+export const getTransactionsHistoryForCurrentMonthAndRUMs = async (month) => {
+  let queryResult;
+  let transactionMonthNumber = 1;
+  let RUMs = [];
+  try {
+    const transactionsHistoryTable = base.getTable(HISTORY_TABLE);
+    queryResult = await transactionsHistoryTable.selectRecordsAsync();
+
+    const formattedHistories = queryResult.records.map(record => ({
+      date: record.getCellValue(HISTORY_DATE_FIELD_ID),
+      RUM: record.getCellValue(HISTORY_RUM_FIELD_ID),
+    }));
+
+    for (const history of formattedHistories) {
+      if (dayjs(history.date).get('month') === month) transactionMonthNumber += 1;
```

Comment:
attention, ici tu ne prend on compte que le mois (et pas l'année). Il faut utiliser IsSame https://day.js.org/docs/en/query/is-same

Reply:
done